turbinia/task_manager.py

```diff
@@ -135,6 +137,30 @@ def _backend_setup(self, *args, **kwargs):
     """
     raise NotImplementedError
 
+  @staticmethod
+  def validate_request_recipe(request_recipe):
+    """Ensure tasks coming from all clients have their recipe checked."""
+    #Load recipe using helper, which will validate.
+    proposed_recipe = request_recipe.get('task_recipes', None)
+    if proposed_recipe:
+      if recipe_helpers.validate_recipe(proposed_recipe):
+        return True
+    return False
+
+  @staticmethod
+  def validate_task_recipe(proposed_recipe, task):
+    """Ensure only allowed parameters are present a given task recipe."""
+    return recipe_helpers.validate_task_recipe(proposed_recipe, task.task_config)
+
+  @staticmethod
+  def get_named_recipe(name, evidence):
```

Comment:
Can we move this to the recipe_helpers module?  We could pass in just the recipe instead of the evidence then.

Reply:
Actually, I don't see this being called from anywhere either?  Should we just remove it?